api/src/main/resources/emails/welcomeemail/WelcomeEmail.html

```diff
@@ -0,0 +1,125 @@
+      box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      margin-bottom: 1rem;
+      cursor: pointer;
+      text-decoration: none;
+    }
+
+    div.card {
+      cursor: default;
+    }
+
+    .bullet-img {
+      height: 31px;
+      width: 31px;
+    }
+
+    .card-text {
+      width: 194px;
+      color: #302C71;
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 19px;
+    }
+
+    .step-img {
+      height: 120px;
+      width: 250px;
+    }
+    .link {
+      display: inline;
+      color: #2691D0;
+    }
+    .button {
+      height: 52px;
+      width: 178px;
+      border-radius: 5px;
+      background-color: #302C71;
+      font-size: 16px;
+      font-weight: 500;
+      line-height: 19px;
+      color: #FFFFFF;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      text-transform: uppercase;
+      text-decoration: none;
+    }
+  </style>
+</head>
+<body>
+  <img src="assets/logo-registration-non-signed-in.svg">
+  <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
+  <div><div class="text user-field">Your User Name: </div><div class="user-info">{{USERNAME-INPUT}}</div></div>
+  <div><div class="text user-field">Your Password: </div><div class="user-info">{{PASSWORD-INPUT}}</div></div>
+  <br>
+  <div class="text">Quick guide on how to get started:</div>
+  <a class="card" href="https://accounts.google.com/signin/oauth/oauthchooseaccount?hd=fake-research-aou.org&client_id=602460048110-5uk3vds3igc9qo0luevroc2uc3okgbkt.apps.googleusercontent.com&as=1ivZqDJMGPEMvWasjAquSQ&destination=http%3A%2F%2Flocalhost%3A4200&approval_state=!ChRqd2tjYnk1b040d3VUMzNpZXBpQRIfczlfc0R4d2dERHdUQU9FaERQUmpBazFLYU13VFBSWQ%E2%88%99AB8iHBUAAAAAWxgpw3y6ICN3aJ_etNsSGfEmZbFNYV2d&xsrfsig=AHgIfE85mPI6c5DfCze82tpt7pbw8-XBRg&flowName=GeneralOAuthFlow">
+    <img class="bullet-img" src="assets/Bullet1@2x.png">
+    <div class="card-text"><div class="link">Sign into google</div> with your new account</div>
+    <img class="step-img" src="assets/Step1@2x.png">
+  </a>
+  <div class="card">
+    <img class="bullet-img" src="assets/Bullet2@2x.png">
+    <div class="card-text">Google will then prompt you to change your password</div>
+    <img class="step-img" src="assets/Step2@2x.png">
+  </div>
+  <a class="card" href="https://myaccount.google.com/security">
+    <img class="bullet-img" src="assets/Bullet3@2x.png">
+    <div class="card-text">Using the Researcher Workbench will require you to <div class="link">set up 2-Step Verification</div></div>
+    <img class="step-img" src="assets/Step3@2x.png">
+  </a>
+  <a class="card" href="https://myaccount.google.com/security">
+    <img class="bullet-img" src="assets/Bullet4@2x.png">
+    <div class="card-text"><div class="link">Check and verify</div> your recovery email address</div>
```

Comment:
This is important for things like forgot your password or similar features.

Reply:
Are you saying that they need to explicitly click "verify" in the Google account UI before they will be able to use password recovery or other features? That would be the only justification I can think of to include this step.

Reply:
Yeah, have confirmed that it appears not to actually get initialized with contact email (tested on my staging account I used before.) 

I'm not entirely certain why. I wonder if recovery email is something separate from contact email.

Reply:
Got it - I didn't see anything about setting up recovery settings in the API docs, so it may be something you need to do via the UI.

We should update the language here, currently it implies that your recovery settings should already be setup and you're just double-checking them.

Reply:
I have, however, figured out how to route directly to recovery email.

Reply:
We cannot set the account recovery email through the gsuite apis (AFAICT). We can set up a custom contact email, but that's a very different thing.